.github/workflows/scale-test-100-gce.yaml

```diff
@@ -226,10 +229,24 @@ jobs:
         with:
           cluster_name: ${{ steps.vars.outputs.cluster_name }}
 
+      - name: Get apiserver loadbalancer address
+        id: apiserver-vars
+        run: |
+          APISERVER_ADDRESS="$(gcloud compute addresses list --filter="addressType:external AND name~'${{ steps.vars.outputs.cluster_name }}'" --format json | jq '.[].address')"
+          CILIUM_INSTALL_APISERVER=" --set k8sServiceHost=${APISERVER_ADDRESS} \
+          --set k8sServicePort=443"
+          echo cilium_install_apiserver=${CILIUM_INSTALL_APISERVER} >> $GITHUB_OUTPUT
+
       - name: Install Cilium
         run: |
-          cilium install --dry-run-helm-values ${{ steps.vars.outputs.cilium_install_defaults }}
-          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+          cilium install --dry-run-helm-values \
+          ${{ steps.vars.outputs.cilium_install_defaults }} \
+          ${{ steps.apiserver-vars.outputs.cilium_install_apiserver }}
```

Comment:
Drive-by comment, an alternative (and slightly simpler) approach used for instance in the scale clustermesh workflow is:

```
    --set k8sServiceHost=api.internal.${{ steps.vars.outputs.cluster_name }} \
    --set k8sServicePort=443 \
```

That mimics how kops configures kube-proxy for instance.

Reply:
Oh cool! Could we try this @thorn3r?

Reply:
Nice! This is much simpler, updated :+1: